assets/services.json

```diff
@@ -1357,12 +1361,14 @@
         "||instagify.com^",
         "||instagmania.com^",
         "||instagor.com^",
+        "||instagr.am^",
```

Comment:
instagr.am, instagram.net
just old domains, which redirect to the main. No need to block it.